### ISSUE TITLE
couldn't use the plugin in English and other problems

### ISSUE THREAD

**Issue by carzamora** [MuiseDestiny/zotero-gpt#39]
Hi, 

I am using Windows 11 and the latest version of Zotero and the plugin; however, it only shows me this window when I start Zotero, if I close it with `Esc` I can not reopen it with` Ctrl + /` and I do not understand how I can configure to display the window in English, if you could incorporate that information in the file readme is very grateful! 

thank you very much for your work! 

![image](https://user-images.githubusercontent.com/5422984/229882286-b0c981a3-ddaa-46b9-b212-866b70986f82.png)

**Comment by MuiseDestiny** [comment]
I am very happy for that an english speaker use this plugin. For the first question, it is Shift but not Ctrl + / due to that Ctrl has a special function in Zotero. Then, you can press a command  tag for a long time to edit its contents incliding tag name, js code and descriptions. Note that press Ctrl + S to save the changes.

**Comment by huachuman** [comment]
Is there any chance of making this plugin display in english?

Many thanks for your consideration and beautiful work.

**Comment by MuiseDestiny** [comment]
> Is there any chance of making this plugin display in english?
> 
> Many thanks for your consideration and beautiful work.

I will do it in the next version.

**Comment by carzamora** [comment]
> I will do it in the next version.

I thought it was already in English because of the labels on the screenshot.

**Comment by carzamora** [comment]
> I am very happy for that an english speaker use this plugin.

I'm not really a native English speaker but a native Spanish speaker! xD several of us are waiting to see it in English or Spanish to see how it works in Zotero! thanks a lot!!!

**Comment by MuiseDestiny** [comment]
> > I will do it in the next version.
> 
> I thought it was already in English because of the labels on the screenshot.

  Yes. In the latest version, I tranform the /help to english.

![image](https://user-images.githubusercontent.com/51939531/230814397-6daf6848-8f3f-4d7b-b637-f3e72b67a617.png)

**Comment by carzamora** [comment]
I still cannot reopen the plugin once it has been closed, if I close it with `Esc` I can not reopen it with `Shift + / ` 

but now I can understand better how the plugin works. Is it 100% necessary to enter the API key and the secret key? because I see that ChatGPT works and answers some things (still in Chinese). So, enter the keys in what helps? sorry for so many questions!

**Comment by carzamora** [comment]
Ok, I was able to solve it! (looking at an image of a Chinese keyboard) 
![image](https://user-images.githubusercontent.com/5422984/231050300-a62d5e88-158a-49ee-b14e-1e723df96c58.png)

I have to press `Shift + ?`    (we have the / in another place on the keyboard)

**Comment by MuiseDestiny** [comment]
> Is it 100% necessary to enter the API key

Yes I recommend you to configure the key, because not configuring the key will use the chatPDF interface, which is preset to Chinese by me. After configuring the key there are no such settings. And chatPDF can only answer 50 questions per day.

> I have to press Shift + ? (we have the / in another place on the keyboard)

Oh, amazing. Sorry, I didn't expect to have this problem.

**Comment by andreifoldes** [comment]
Could it be possible to allow for customizing the hotkey/shortcut for the wake up command? To associate it with any preferred button instead of "Shift +? | Shift + /"?

**Comment by MuiseDestiny** [comment]
> Could it be possible to allow for customizing the hotkey/shortcut for the wake up command? To associate it with any preferred button instead of "Shift +? | Shift + /"?

Ok. Next version.

**Comment by 31SFX4** [comment]
Can you please add a way to open it with the mouse (right click, context menu)? In addition to customize the shortcut.

I have a German keyboard and also can't use the shift-/ shortcut. So I can only use it when I change the keyboard to US.

**Comment by MuiseDestiny** [comment]
> Can you please add a way to open it with the mouse (right click, context menu)? In addition to customize the shortcut.您能否添加一种用鼠标打开它的方法（右键单击，上下文菜单）？除了自定义快捷方式。
> 
> I have a German keyboard and also can't use the shift-/ shortcut. So I can only use it when I change the keyboard to US.我有一个德语键盘，也不能使用 shift-/ 快捷方式。所以我只能在将键盘更改为美国时使用它。

Now `Ctrl + /`. well, I will add a GPT icon to open it in the next version.

**Comment by 31SFX4** [comment]
Thank you!

**Comment by BlueDrink9** [comment]
English users will need to change the gpt prompt text too I think. #324 has translations

**Comment by Cynane23** [comment]
Hi there is just random time that it will appear in chinese but the next word is english. Is there a way to configure it that all characters are in english not the swapping back and forth?

**Comment by Calorion** [comment]
> Hi there is just random time that it will appear in chinese but the next word is english. Is there a way to configure it that all characters are in english not the swapping back and forth?

Edit the System Prompt and change the output language to English.

**Comment by Cynane23** [comment]
> Edit the System Prompt and change the output language to English.

Sorry I'm not really a coder and I don't know how to do this. I use this more as a (extremely useful) tool to help me annotate readings for my law course. Could you please instruct me on how I can do this?

**Comment by Calorion** [comment]
> Sorry I'm not really a coder and I don't know how to do this. I use this more as a (extremely useful) tool to help me annotate readings for my law course. Could you please instruct me on how I can do this?

There’s no coding. Go to the preference pane of the plugin, click “Edit” next to “System Prompt,” and the rest should be obvious.

If it’s not, I can provide a screenshot.

**Comment by MuiseDestiny** [comment]
<img width="1017" height="713" alt="Image" src="https://github.com/user-attachments/assets/7baaf40e-b5bd-4638-87c5-9b0f896c60e6" />
